ckanext/datagovtheme/helpers.py

```diff
@@ -462,10 +462,10 @@ def get_bureau_info(bureau_code):
       bureau_code = bureau_code[0]
 
     filepath = ckan_tmp_path + '/logos/'
-    filename = filepath + 'bureau.csv'
+    filename = filepath + 'bureau-sample.csv'
     url = config.get('ckanext.geodatagov.bureau_csv.url', '')
     if not url:
-        url = config.get('ckanext.geodatagov.bureau_csv.url_default', '')
+        url = config.get('ckanext.geodatagov.bureau_csv.url_default', BUREAU_CODE_URL)
```

Comment:
```suggestion
        BUREAU_CODE_URL = 'https://project-open-data.cio.gov/data/omb_bureau_codes.csv'
        url = config.get('ckanext.geodatagov.bureau_csv.url_default', BUREAU_CODE_URL)
```
Yo still need this value inside this ext. The .env file will work just for the CircleCI env, and not when you install the ext